src/unit_tests/kubernetes_api_objects/container.rs

```diff
@@ -216,17 +180,48 @@ pub fn test_clone(){
 
 #[test]
 #[verifier(external)]
-pub fn test_overwrite_resources(){
-    let mut container = Container::default();
-    let mut resources = ResourceRequirements::default();
-    let mut requests = StringMap::new();
-    requests.insert(new_strlit("cpu").to_string(), new_strlit("100m").to_string());
-    resources.set_requests(requests);
-    container. overwrite_resources(Some(resources.clone()));
-    assert_eq!(resources.into_kube(), container.into_kube().resources.unwrap());
-    let mut container_2 = Container::default();
-    let resources_2 = None;
-    container_2.overwrite_resources(resources_2);
-    assert_eq!(None, container_2.into_kube().resources);
+pub fn test_kube() {
+    let container = Container::from_kube(deps_hack::k8s_openapi::api::core::v1::Container {
```

Comment:
@ZichengMa Can you change it to 
```
let kube_container = deps_hack::k8s_openapi::api::core::v1::Container { ... };
let container = Container::from_kube(kube_container.clone());
assert_eq!(container.into_kube(), kube_container);
```
and apply the pattern to all the following tests?
So that you don't need to duplicate the big chunk of struct construct.

Reply:
@ZichengMa @marshtompsxd perhaps convert it to an issue so we don't forget it.

Reply:
https://github.com/vmware-research/verifiable-controllers/issues/399